ouroboros-consensus/src/ouroboros-consensus/Ouroboros/Consensus/Mempool/Update.hs

```diff
@@ -189,7 +189,17 @@ pureTryAddTx cfg txSize wti tx is
   , let curTotalRefScriptSize = isTotalRefScriptSize is
         newTxRefScriptSize    = txRefScriptSize cfg (isLedgerState is) tx
         maxTotalRefScriptSize = 1024 * 1024 -- 1MiB
-  , curTotalRefScriptSize + newTxRefScriptSize Prelude.<= maxTotalRefScriptSize
+        mempoolStaysBelowCapacity =
+          curTotalRefScriptSize + newTxRefScriptSize Prelude.<= maxTotalRefScriptSize
```

Comment:
Just a note, perhaps worth including as a comment.

There is potential for overflow in this check, which would cause a false `False` in this first disjunct. But it could in practice only overflow if `newTxRefScriptSize` were huge, which would set the second disjunct to `True`. So it's fine as-is.

Reply:
Changed accordingly :+1: